test/powershell/Language/Scripting/ActionPreference.Tests.ps1

```diff
@@ -33,22 +33,23 @@ Describe "Tests for (error, warning, etc) action preference" -Tags "CI" {
             $error.Count | Should -BeExactly $errorCount
         }
 
-        It 'action preference of Ignore cannot be set as a preference variable' {
-            $e = {
-                $GLOBAL:errorActionPreference = "Ignore"
-                Get-Process -Name asdfasdfasdf
-            } | Should -Throw -ErrorId 'System.NotSupportedException,Microsoft.PowerShell.Commands.GetProcessCommand' -PassThru
-            $e.CategoryInfo.Reason | Should -BeExactly 'NotSupportedException'
+        It 'action preference of Ignore can be set as a preference variable using a string value' {
+            Remove-Variable -Name ErrorActionPreference -Scope Global -Force
+            $GLOBAL:errorActionPreference = "Ignore"
```

Comment:
while unlikely, it's possible that the setting could leak, I'd be happier if this was in a `try { } finally { }`

Reply:
Agreed, done local, will commit soon.